src/applications/mhv-secure-messaging/containers/InterstitialPage.jsx

```diff
@@ -36,13 +36,19 @@ const InterstitialPage = props => {
           reply.
         </p>
 
-        <va-button
-          uswds
+        <span className="sr-only" id="continue-content-change-message">
+          Page content will change.
+        </span>
+
+        <button
```

Comment:
Is there a reason for not using `<va-button>` here?

Reply:
@powellkerry 👋🏻  While @KyleCardwell is in-n-out this week, I can try to elaborate. This button basically serves as an acknowledgment a user has to accept before starting a message. Which then changes the page from this "Acknowledgement" to a `compose a message form`. Per `a11y` feedback, we have to give screen reader users a specific feedback that `Page content will change`. We were advised not to change `aria-label` on a button (as this will not be compliant and usable with voice command users), but rather add `aria-describedby`. However, `<va-button>` does not seem to be friendly with `aria-describedby` attribute no matter what we tried.

Reply:
@powellkerry thank you 👍🏻